packages/dx-grid-core/src/plugins/table-column-visibility/helpers.js

```diff
@@ -0,0 +1,4 @@
+import { TABLE_DATA_TYPE } from '../table/constants';
+
+export const isEmptyMessageShow = tableColumns =>
```

Comment:
Maybe use the 'isEmptyMessageShown' name here?

Reply:
I'd prefer something like `tableDataColumnsExist`. In this case, we can use it as follows:
```js
<TemplateConnector>
  {({ tableColumns }) =>
    (tableDataColumnsExist(tableColumns)
    ? <TemplatePlaceholder />
    : <EmptyMessage
      getMessage={getMessage}
      {...params}
    />
  )}
</TemplateConnector>
```